tuf/conf.py

```diff
@@ -54,6 +54,10 @@
 # be saved to 'LOG_FILENAME'
 LOG_FILENAME = 'tuf.log'
 
+# This is the format/encoding of metadata that TUF will use. The options are
+# 'json' and 'der'. DER is an encoding of ASN.1 used for the Uptane project.
+METADATA_FORMAT = 'json' # 'der'
```

Comment:
Do we expect the format of all metadata on a repository to be equal?  For instance, must it all be JSON, or all DER?  No mixing of formats?